server/src/main/java/com/testsigma/service/StorageConfigService.java

```diff
@@ -22,4 +26,20 @@ public StorageConfig update(StorageConfig storageConfig) {
     return storageConfigRepository.save(storageConfig);
   }
 
+    public boolean validateCredentials(StorageConfig storageConfigReq) {
+      switch (storageConfigReq.getStorageType()) {
+        case AWS_S3:
+          if (storageConfigReq.getAwsAccessKey()==null && storageConfigReq.getAwsSecretKey()==null) return true;
```

Comment:
@shanmukha2000 get instance using storageFaktory and move respective logic to services.